Tests/configure_and_test_integration_instances.py

```diff
@@ -626,16 +626,73 @@ def set_marketplace_gcp_bucket_for_build(client, prints_manager, branch_name, ci
     prints_manager.add_print_job(installed_content_message, print_color, 0, LOG_COLORS.GREEN)
 
     # make request to update server configs
+    server_configuration = {
+        'content.pack.verify': 'false',
+        'marketplace.initial.sync.delay': '0',
+        'content.pack.ignore.missing.warnings.contentpack': 'true',
+        'marketplace.bootstrap.bypass.url':
+            'https://storage.googleapis.com/marketplace-ci-build/content/builds/{}/{}'.format(
+                branch_name, ci_build_number)
+
+    }
+    error_msg = "Failed to set GCP bucket server config - with status code "
+    response_data, status_code, _ = update_server_configuration(client, server_configuration, error_msg)
+
+
+def set_docker_hardening_for_build(client, prints_manager):
+    """Sets docker hardening configuration
+
+    Args:
+        client (demisto_client): The configured client to use.
+        prints_manager (ParallelPrintsManager): Print manager object
+
+    Returns:
+        None
+    """
+    host = client.api_client.configuration.host
+    installed_content_message = \
+        '\nMaking "POST" request to server - "{}" to set docker hardening server configuration.'.format(host)
+    prints_manager.add_print_job(installed_content_message, print_color, 0, LOG_COLORS.GREEN)
+
+    # make request to update server configs
+    server_configuration = {
+        'docker.cpu.limit': '1.0',
+        'docker.run.internal.asuser': 'true',
+        'limit.docker.cpu': 'true',
+        'python.pass.extra.keys': '--memory=1g##--memory-swap=-1##--pids-limit=256##--ulimit=nofile=1024:8192'
+    }
+    error_msg = "Failed to set docker hardening server config - with status code "
+
+    response_data, status_code, _ = update_server_configuration(client, server_configuration, error_msg)
+
+
+def get_pack_ids_to_install():
+    with open('./Tests/content_packs_to_install.txt', 'r') as packs_stream:
+        pack_ids = packs_stream.readlines()
+        return [pack_id.rstrip('\n') for pack_id in pack_ids]
```

Comment:
move back to the original location (above `main`, below `update_server_configuration`)

Reply:
done